themes/dark/discord.css

```diff
@@ -18,7 +18,7 @@ body {
 	--roundness: 1; /* How round is this theme? Applied automatically to borders. Unitless */
 
 	/* Fonts! */
-	font-family: PTSans, sans-serif;
+	font-family: '', sans-serif;

```

Comment:
You cannot simply omit a font...?

Reply:
It's works for me, and shows the font that I want.

Reply:
Or i'm going to do something for that